src/components/Task.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div id="task">
+   <h1>ToDo リスト</h1>
+
+    <input type="radio" name="condition" value="すべて" v-model="picked" checked>すべて
+    <input type="radio" name="condition" value="作業中" v-model="picked">作業中
+    <input type="radio" name="condition" value="完了" v-model="picked">完了
+
+    <table :class="table">
+      <thead>
+        <tr>
+          <th>ID</th>
+          <th>コメント</th>
+          <th>状態</th>
+        </tr>
+      </thead>
+      <tbody v-for="todos in filterStatus" :key="todos.id">
+        <tr>
+          <td>{{todos.id}}</td>
+          <td>{{todos.content}}</td>
+          <td>
+            <button @click="changeStatus(todos.id)">{{todos.status}}</button>
+            <button @click="removeTodo(todos.id)">削除</button>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h2>新規タスクの追加</h2>
+    <input type="text" name="newTask" v-model="text">
+    <button @click="addTodo">追加</button>
+  </div>
+</template>
+
+<script>
+export default{
+  data: function() {
+    return {
+      todo: {
+        id: '',
+        contnet: '',
+        status: ''
+      },
+      tasks: [],
+      picked: 'すべて',
+      text: '',
+      count: 0,
+      /*CSSクラス*/
+      table: 'table'
+    }
+  },
+  computed: {
+    filterStatus: function() {
+      let filterTasks = null;
+      if (this.picked === 'すべて') {
+        filterTasks = this.tasks;
+      } else if (this.picked === '作業中') {
+        filterTasks = this.tasks.filter( function(list) {
+          return list.status === '作業中';
+        });
+      }else if (this.picked === '完了') {
+        filterTasks = this.tasks.filter( function(list) {
+          return list.status === '完了';
+        });
+      }
+      return filterTasks;
+    }
+  },
+  methods: {
+    addTodo: function() {
+      if (this.text === '') {

```

Comment:
空文字はfalsyな扱いになるので、条件を以下のように記載できます。
```suggestion
      if (!this.text) {
```